src/components/CurrencyInputPanel/SwapCurrencyInputPanel.tsx

```diff
@@ -312,7 +312,13 @@ export default function SwapCurrencyInputPanel({
           <FiatRow>
             <RowBetween>
               <LoadingOpacityContainer $loading={loading}>
-                {fiatValue && <FiatValue fiatValue={fiatValue} priceImpact={priceImpact} />}
+                {fiatValue && (
+                  <FiatValue
+                    fiatValue={fiatValue}
+                    priceImpact={priceImpact}
+                    testId={`fiat-value-${id === InterfaceSectionName.CURRENCY_INPUT_PANEL ? 'input' : 'output'}-panel`}
```

Comment:
Is it necessary to name the input and output panels differently? An alternative might be to select both and index them:
```
// input panel
cy.get(getTestSelector('fiat-value-panel')).eq(0)
// output panel
cy.get(getTestSelector('fiat-value-panel')).eq(1)
```
I think it's good to limit the business logic in components as much as possible.

Reply:
Additionally, would this allow us to remove `testId` as a prop and just statically apply it inside of `FiatValue`, instead of passing it down as a dynamic value from `SwapCurrencyInputPanel`?

Reply:
```suggestion
                    testId={`fiat-value-${id}`}
```